www/css/style.css

```diff
@@ -135,4 +141,17 @@ background:url('../img/contact.background.jpg') no-repeat center center fixed;
   border: 0px;
 }
 
+.booking-button {
+  margin-top: 0px;
+  margin-bottom: 0px;
+  border: 0px;
+  padding: 10px 10px 10px 10px;
+}
+
+h3 {
+  font-family: 'Julius Sans One', sans-serif;
+}
 
+.input-label {
+  font-family: 'Julius Sans One', sans-serif;
+}
```

Comment:
If you guys are changing the font family everywhere, can't you just do it on `body`?